charts/scalardb-graphql/ci/ct-values.yaml

```diff
@@ -0,0 +1,2 @@
+scalarDbGraphQlConfiguration:
+  namespaces: ct
```

Comment:
`namespaces` is a required config item for the scalardb-graphql chart.

chart-testing uses the glob pattern `<chart root>/ci/*-values.yaml` to find custom values files.

https://github.com/helm/chart-testing/blob/main/doc/ct_install.md#ct-install

> Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is installed and tested for each of these files. If no custom values file is present, the chart is installed and tested with defaults.